### ISSUE TITLE
Tiny bundled wire not working with circuits within circuits

### ISSUE THREAD

**Issue by SageStarCodes** [amadornes/SCM-Localization-Issues#304]
I had this circuit: (https://gist.github.com/anonymous/73bed3fb5086d8fe5814eae8cb8e63f8)
![2017-01-02_22 02 06](https://cloud.githubusercontent.com/assets/1764903/21596130/1e0e7db0-d137-11e6-9318-2b4686d22801.png)

As part of a bigger circuit outputting all the activity on the lines to a single color acting as an activity monitor for a watchdog circuit, I tried putting this circuit onto another circuit a was testing it with the levers and I noticed that when I would flip one of the levers nothing would happen.
The mod version is 1.0.4, in the most up-to-date public release of the "FTB Presents Direwolf20 1.10" modpack

**Comment by Nonsanity** [comment]
v1.0.4 for Minecraft 1.10 (Quantus modpack on Curse)

I have some further info on this issue I've detailed in a short explanatory video here:
               https://www.youtube.com/watch?v=n2lYWELwe4Y
It is unlisted, so you won't be able to search for it. I noticed that a similar bug report was close on Nov 14th as "already fixed," however the problem exists in 1.0.4 still.

This bug concerns bundled cable outputs from embedded circuits. Once a bundled cable output from a circuit has a signal strength greater than what it had before, it will retain it permanently.

My guess is that the exiting bundled cable is feeding the same charge back in, since the bundled cables don't have the same one-way interface at circuit edges that the bare wires do. But that's just a wild guess from my observations.

Let me know if I can help solve this in any way.

**Comment by dokranke** [comment]
I agree with nonsanity, the bug still exist. Only tiny redstone piles should work for i/o on integrated circuits for now.

Also integrated multi-level circuits tend to crash often due to the redstone impregnated sticks
